README.md

```diff
@@ -22,13 +22,20 @@ The [code](https://github.com/runame/laplace-redux) to reproduce the experiments
 
 ## Setup
 
-We assume `python3.8` since the package was developed with that version.
+For full compatibility, install this package in a fresh virtual env.
+We assume Python >= 3.8 (last tested on Python 3.10.12).
 PyTorch version 2.0 and up is also required for full compatibility.
 To install laplace with `pip`, run the following:
 ```bash
 pip install laplace-torch
 ```
 
+
+Note that, if you would like to use ASDL as a backend, please install it via the following, for full compatibility:
```

Comment:
Can you make more precise for what kind of compatibility? This is presumably necessary only for LLM stuff? We should try to merge these changes into ASDL if possible.

Reply:
The PR is open. I think this is non-blocking. We can update the `setup.cfg` once Kazuki merged it.

Reply:
https://github.com/kazukiosawa/asdl/pull/52